app/models/industry.rb

```diff
@@ -1,4 +1,5 @@
 class Industry < ActiveRecord::Base
   has_many :sectors, :dependent => :destroy
   validates :name, presence: true, uniqueness: true
+  validates :protege_id, presence: true, uniqueness: true
```

Comment:
You've added this new property to this model but haven't updated [the spec](https://github.com/GovWizely/industry_mapping/blob/master/spec/models/industry_spec.rb).